using-timescaledb/data-retention.md

```diff
@@ -20,6 +27,41 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
+### Data Retention and Continuous Aggregates
+
+The function `drop_chunks` can be used to control data retention for
+the continuous aggregate as well, but if you have a continuous
+aggregate defined on the table you need to provide a value for the
+`cascade_to_materialization`. Not giving a value when there is a
+continuous aggregate defined on a hypertable will generate an error.
```

Comment:
> Not giving a value when there is a continuous aggregate defined on a hypertable will generate an error

This sounds super-odd to me. I would double-check that this is the case.

Just saying.